Argo/Types/Decoded/Decoded.swift

```diff
@@ -28,6 +56,14 @@ public extension Decoded {
     }
   }
 
+  /**
+    Convert an `Optional` into a `Decoded` value.
+
+    If the provided optional is `.Some`, this method extracts the value and
+    wraps it in `.Success`. Otherwise, it returns a `.TypeMismatch` error
+
+    - returns: The provided `Optional` value transformed into a `Decoded` value
```

Comment:
Oh actually none of the `parameter` lines end in a period and of the `returns` docs, only line 83 ends in a period. So this is fine, but you may want to change line 83.